beacon/beacon.cabal

```diff
@@ -33,8 +33,10 @@ executable beacon
     , containers
     , directory
     , extra
+    , MissingH
```

Comment:
I think this can now be removed. Also, what about adding `-Wunused-packages`? (just noticed that currently, there are no warnings at all :thinking:)